tox.ini

```diff
@@ -41,17 +41,15 @@ passenv =
   AZURE_PROD_STORAGE_ACCESS_KEY
   AZURE_DEV_STORAGE_ACCOUNT
   AZURE_DEV_STORAGE_ACCESS_KEY
-  AWS_TESTING_ACCESS_KEY_ID
-  AWS_TESTING_SECRET_ACCESS_KEY
-  AWS_PROD_ACCESS_KEY_ID
-  AWS_PROD_SECRET_ACCESS_KEY
-  AWS_DEV_ACCESS_KEY_ID
-  AWS_DEV_SECRET_ACCESS_KEY
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
+  AWS_SESSION_TOKEN
+  AWS_PROFILE
+  AWS_CONFIG_FILE
   AWS_DEFAULT_REGION
 deps =
   dnspython
   pytest
-  pytest-catchlog
```

Comment:
Why is this removed?

Reply:
because when you run pytest/tox you see this:
`pytest-catchlog plugin has been merged into the core, please remove it from your requirements`
https://teamcity.mesosphere.io/viewLog.html?buildId=931198&buildTypeId=DcOs_Open_Build_Tox&tab=buildLog&_focus=1523#_state=268